packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -690,12 +690,28 @@ export const makeInnerVault = (
     makeAdjustBalancesInvitation,
     makeCloseInvitation,
     makeTransferInvitation: () => {
-      const { outerUpdater } = state;
+      // Bring the debt snapshot current for the final report before transfer
+      updateDebtSnapshot(getCurrentDebt());
+      const {
+        outerUpdater,
+        debtSnapshot: debt,
+        interestSnapshot: interest,
+        phase,
+      } = state;
       if (outerUpdater) {
         outerUpdater.finish(snapshotState(VaultPhase.TRANSFER));
         state.outerUpdater = null;
       }
-      return zcf.makeInvitation(makeTransferInvitationHook, 'TransferVault');
+      const transferState = {
```

Comment:
The `priceAuthority` and `mint` are in there for durability. They should likely be all folded into a single manager reference. 

The notifier should not be in the transfer state because the previous one is closed and the new one won't be created until the transfer is done.